app/helpers/ontology_helper.rb

```diff
@@ -13,4 +13,53 @@ def download_path(resource)
   def show_evaluate?
     show_oops? #|| show_foo?
   end
+
+  def status_tag(resource)
```

Comment:
This method seems to be unused (a `grep` couldn't find any other occurence other than the definition itself).